model/model.go

```diff
@@ -10,9 +10,9 @@ const (
 )
 
 type Card struct {
-	Suit Suit `protobuf:"varint,1,req,name=suit,proto3" json:"suit" bson:"suit"` //nolint:lll
+	Suit Suit `protobuf:"varint,1,req,name=suit,proto3" json:"s" bson:"s"` //nolint:lll
```

Comment:
I made these changes because our json/bson tags should be minimal to reduce size (not _that_ big of a deal, but we shouldn't be bloating things if we can afford it). It will break anything you currently have persisted in mongodb.